src/vm/gcheaphashtable.h

```diff
@@ -0,0 +1,144 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef GCHEAPHASHTABLE_H
+#define GCHEAPHASHTABLE_H
+
+class GCHeapHashObject;
+
+template <bool remove_supported>
```

Comment:
A nit - can you please consider using camel case naming convention instead of the underscores for the sake of consistency with the other code.

Reply:
done.